README.md

```diff
@@ -17,6 +17,7 @@ Metabase is the easy, open source way for everyone in your company to ask questi
 - Send data to Slack or email on a schedule with [Pulses](https://metabase.com/docs/latest/users-guide/10-pulses.html)
 - View data in Slack anytime with [MetaBot](https://metabase.com/docs/latest/users-guide/11-metabot.html)
 - [Humanize data](https://metabase.com/docs/latest/administration-guide/03-metadata-editing.html) for your team by renaming, annotating and hiding fields
+- What happend with my data? Receive [alerts](https://www.metabase.com/docs/latest/users-guide/15-alerts.html)
```

Comment:
```suggestion
- See changes in your data with [alerts](https://www.metabase.com/docs/latest/users-guide/15-alerts.html)
```
Thanks for adding this. I think we'll want to keep this in line with our other feature bullets and not use a question here. Could you also please sign the CLA for us?